### PR TITLE
apidump: Append file extension is not present

### PR DESCRIPTION
**Pull request by charles-lunarg**
If the filename provided in the settings file does not end with the corresponding extension for the output type, api dump will append it.

### CODE REVIEW

File: layersvt/api_dump.h [hunk 1]
```diff
@@ -475,43 +475,69 @@ class ApiDumpSettings {
...
-                if (vkuHasLayerSetting(layerSettingSet, kSettingsKeyLogFilename)) {
-                    vkuGetLayerSettingValue(layerSettingSet, kSettingsKeyLogFilename, filename_string);
-                    if (filename_string.empty()) {
-                        filename_string = "vk_apidump.txt";
-                    }
-                }
+            if (output_format == ApiDumpFormat::Html) {
+                filename_string = "vk_apidump.txt";
+            } else if (output_format == ApiDumpFormat::Json) {
+                filename_string = "vk_apidump.txt";
```
Comment (Tony-LunarG) [inline layersvt/api_dump.h:503]: Should all three of these be the same?  Should html end with .html?

**Review by Tony-LunarG — COMMENTED** [review]
Question about doing the same assign independent of output_format

File: layersvt/api_dump.h (hunk 1 above)
Comment (charles-lunarg) [reply layersvt/api_dump.h:503]: Thats a copy paste error! Good catch.

**Review by Tony-LunarG — APPROVED** [review]
LGTM

**Pull request merged**